docs/build/javascript/quick-start.md

```diff
@@ -0,0 +1,144 @@
+# Quickstart
+Learn the basics by setting up and interacting with the [Ceramic CLI](./cli/installation.md)
+
+!!! warning ""
+    **Want an even faster way to try Ceramic? Visit the [Playground Demo App](https://playground.ceramic.dev){:target="_blank"} to test the full stack of Ceramic Components in the browser.
+
+## **1. Install the Client**
+
+Visit the [Ceramic Client](./javascript/installation.md) page for instructions on how to quickly install the Client.
+
+## **2. Create a stream**
+Creating streams is dependent on the StreamType of the record. More details can be found in our [StreamTypes Overview](../streamtypes/overview.md). A basic sample can be found below.
+
+=== "Command"
+
+    ```JavaScript
+    import { TileDocument } from '@ceramicnetwork/stream-tile'
+
+    const schemaId = 'k3y52l7qbv1fry1fp4s0nwdarh0vahusarpposgevy0pemiykymd2ord6swtharcw' // this is our Basic Profile Schema.
+    
+    const content = {
```

Comment:
I don't think this will pass validation of the basic profile schema.  If we're going to use a real concrete schema in the example we should make sure it actually works with the example data

Reply:
I'm not sure how big is the minimal profile that will be accepted by the Basic Profile schema.  If it's small you could update the test data to match the schema, but if it's large I also think it'd be fine to just not set a schema in this example.

Reply:
Oops, wrote the content before adding the schema. 

Regarding not needing the schema, I thought the same thing initially but decided to use the basic schema so that someone following along could actually create their own stream easily. I'd rather provide all of the useful information where it's needed (unless it's explained in a previous step).

Reply:
for the record, schemas are optional in TileDocuments so you don't *need* to set a schema to create a Stream.  But I see you updated the content to match the basic profile schema, and that seems fine for this example.